Dockerfile

```diff
@@ -10,9 +10,12 @@ COPY . /app
 
 # install dependencies
 RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
By the way, good hack to remember: Docker builds image layer by layer. If some layer is changed then all subsequent layers are rebuilt too.
You copy whole code first (COPY . /app) and install the requirements then.
So if any code files was changed then all requirements must be reinstalled too - that takes a lot of time.

Alternatively, you can install the requirements first and copy the code then:

```
WORKDIR /app
RUN pop install gunicorn
COPY requirements.txt .
RUN pop install -r requirements.txt
COPY . /app
```
In this way image rebuild will be instant if only source code was changed, the layer with python packages will be cached and reused

Reply:
Good, fixed